### PR TITLE
ie10 issue roundup part one (bug 1135213)

### PR DESCRIPTION
**Pull request by spasovski**
Some issues still remain but I'll file separate bugs for those. Not sure how much time we should spend on these since Edge seems like it will cover all the older flexbox/event issues etc.

### CODE REVIEW

**Comment by sherlockedci** [comment]
[ The Adventure of the Yellow Build](http://sherlocked.dev.mozaws.net/builds/67564631)

File: src/media/js/elements/header.js [hunk 1]
```diff
@@ -26,10 +26,9 @@
         - Toggle all header children closed.
 */
 define('elements/header',
-    ['core/element_utils', 'core/z', 'document-register-element', 'jquery',
+    ['core/element_utils', 'core/z', 'document-register-element',
      'underscore'],
-    function(eUtils, z, dre, $,
-             _) {
+    function(eUtils, z, dre, _) {
```
Comment (mstriemer) [inline src/media/js/elements/header.js]: I think the `_` should stay where it was. These normally match the dependencies format,

Comment (ngokevin) [inline src/media/js/elements/header.js]: yeah. it should match the breaks above for AMD.

**Comment by ngokevin** [comment]
r+

**Pull request merged**